tests/rptest/tests/pandaproxy_test.py

```diff
@@ -1238,3 +1243,41 @@ def test_password_change(self):
         # Put the original password back incase future changes to the
         # teardown process in RedpandaService relies on the superuser
         admin.update_user(super_username, super_password, super_algorithm)
+
+
+class PandaProxyAutoAuthTest(PandaProxyEndpoints):
+    """
+    Testpandaproxy against a redpanda cluster with Auto Auth enabled.
+    """
+    def __init__(self, context):
+        security = SecurityConfig()
+        security.kafka_enable_authorization = True
+        security.endpoint_authn_method = 'sasl'
+        security.auto_auth = True
+
+        super(PandaProxyAutoAuthTest, self).__init__(context,
+                                                     security=security)
+
+    @cluster(num_nodes=3)
+    def test_get_topics(self):
+        nodes = self.redpanda.nodes
+        node_count = len(nodes)
+        restart_node_idx = 0
+
+        def check_connection(hostname: str):
+            result_raw = self._get_topics(hostname=hostname)
+            self.logger.info(result_raw.status_code)
+            self.logger.info(result_raw.json())
+            assert result_raw.status_code == requests.codes.ok
+            assert result_raw.json() == []
```

Comment:
similar comment to the other pr. would be nice to have the negative case, as well as enabling auth on the other tests.

Reply:
I agree. I'm struggling to think of a way to test it in ducktape that isn't just the negative case of the SASl test.

But the main reason to run the entire test suite is to test the automagic ACLs. I'll add more tests.

Reply:
cool. feel free to merge and add more tests in follow up